infrastructure/server/flyway.tf

```diff
@@ -229,7 +229,10 @@ data "aws_lambda_invocation" "invoke_flyway_migration" {
           "client_id_prod_fom_oidc_client" : "${aws_cognito_user_pool_client.prod_fom_oidc_client.id}",
           "client_id_dev_spar_oidc_client" : "${aws_cognito_user_pool_client.dev_spar_oidc_client.id}",
           "client_id_test_spar_oidc_client" : "${aws_cognito_user_pool_client.test_spar_oidc_client.id}",
-          "client_id_prod_spar_oidc_client" : "${aws_cognito_user_pool_client.prod_spar_oidc_client.id}"
```

Comment:
The client IDs from Cognito need to be added as placeholder values to create the records in app_fam.application_client for the forest client application entries.